src/main/java/kr/motd/maven/os/Detector.java

```diff
@@ -54,6 +69,11 @@ protected void detect(Properties props) throws DetectionException {
                 throw new DetectionException("unknown os.arch: " + osArch);
             }
         }
+
+        for (String like : getOsLike(detectedName)) {
```

Comment:
Shouldn't this run only when the current OS is Linux or freedesktop.org-compatible one? For example, this should not run on Windows and OS X.

Also, I wonder what value we should use for such OSes. `generic`?